prac_06/car.py

```diff
@@ -0,0 +1,36 @@
+"""CP1404/CP5632 Practical - Car class example."""
+
+
+class Car:
+    """Represent a Car object."""
+
+    def __init__(self, name="", fuel=0):
+        """Initialise a Car instance.
+
+        fuel: float, one unit of fuel drives one kilometre
+        name: string, cars name
+        """
+        self.name = name
+        self.fuel = fuel
+        self._odometer = 0
+
+    def __str__(self):
```

Comment:
I noticed that in some demos, Lindsay has added a docstring for the __str__() method, like:
"""Return string representation of data in a Car object."""